### PR TITLE
Remove fluent-bit rawConfig mentions after upgrading to new fluent-bit chart

### PR DESCRIPTION
**Pull request by pmalek-sumo**
###### Testing performed

- [ ] ci/build.sh
- [ ] Redeploy fluentd and fluentd-events pods
- [ ] Confirm events, logs, and metrics are coming in

### CODE REVIEW

File: deploy/docs/Best_Practices.md [hunk 1]
```diff
@@ -246,16 +246,17 @@ you will need to ensure the logs get mounted to the host so fluent-bit can be
...
 
 Example:
-In `values.yaml` in the `fluent-bit.rawConfig` section, you have to add
-a new input specifying the file path, eg.
+In `values.yaml` in the `fluent-bit.config.input` section, you have to add
+a new `INPUT` specifying the file path (retaining the remaining part of `input`
+config), e.g.:
 
-```bash
+```
```
Comment (sumo-drosiek) [inline deploy/docs/Best_Practices.md:253]:
```suggestion
```yaml
fluent-bit:
  config:
    input: |-
      # Here should be copied original fluent-bit.config.input
```

and indentation for the section

File: deploy/helm/sumologic/README.md [hunk 2]
```diff
@@ -182,7 +182,6 @@ Parameter | Description | Default
...
...lter _SYSTEMD_UNIT=locksmithd.service Systemd_Filter _SYSTEMD_UNIT=logrotate.service Systemd_Filter _SYSTEMD_UNIT=lvm2-monitor.service Systemd_Filter _SYSTEMD_UNIT=mdmon.service Systemd_Filter _SYSTEMD_UNIT=nfs-idmapd.service Systemd_Filter _SYSTEMD_UNIT=nfs-mountd.service Systemd_Filter _SYSTEMD_UNIT=nfs-server.service Systemd_Filter _SYSTEMD_UNIT=nfs-utils.service Systemd_Filter _SYSTEMD_UNIT=node-problem-detector.service Systemd_Filter _SYSTEMD_UNIT=ntp.service Systemd_Filter _SYSTEMD_UNIT=oem-cloudinit.service Systemd_Filter _SYSTEMD_UNIT=rkt-gc.service Systemd_Filter _SYSTEMD_UNIT=rkt-metadata.service Systemd_Filter _SYSTEMD_UNIT=rpc-idmapd.service Systemd_Filter _SYSTEMD_UNIT=rpc-mountd.service Systemd_Filter _SYSTEMD_UNIT=rpc-statd.service Systemd_Filter _SYSTEMD_UNIT=rpcbind.service Systemd_Filter _SYSTEMD_UNIT=set-aws-environment.service Systemd_Filter _SYSTEMD_UNIT=system-cloudinit.service Systemd_Filter _SYSTEMD_UNIT=systemd-timesyncd.service Systemd_Filter _SYSTEMD_UNIT=update-ca-certificates.service Systemd_Filter _SYSTEMD_UNIT=user-cloudinit.service Systemd_Filter _SYSTEMD_UNIT=var-lib-etcd2.service Max_Entries 1000 Read_From_Tail true @INCLUDE fluent-bit-output.conf`
```
Comment (sumo-drosiek) [inline deploy/helm/sumologic/README.md:185]: Shouldn't we add `fluent-bit.config.input` etc instead?

Comment (pmalek-sumo) [reply deploy/helm/sumologic/README.md:185]: We could (should?) but how do you suggest to put the multiline yaml code snippet in the table?
https://github.com/SumoLogic/sumologic-kubernetes-collection/blob/13438eeb5112b8283d89dc9b91676dc750cccb5b/deploy%2Fhelm%2Fsumologic%2Fvalues.yaml#L828-L894

Comment (pmalek-sumo) [reply deploy/helm/sumologic/README.md:185]: This answer on SO says that it's not possible to add multiline code snippet in a table cell in markdown:

https://stackoverflow.com/a/28514979

**Review by sumo-drosiek — APPROVED** [review]

**Pull request merged**